File: A2Z_sheet/Array/Easy/03_check_if_array_is_sorted_and_rotated/solution.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        count = 0
        for i in range(1, len(nums)):
            if nums[i] < nums[i - 1]:
                count += 1
        if count > 1:
            return False
        return True if count == 0 else nums[0] >= nums[-1]

File: A2Z_sheet/Array/Easy/03_check_if_array_is_sorted_and_rotated/solution.go
func check(nums []int) bool {
    count := 0
    for i := 1; i<len(nums); i++ {
        if(nums[i] < nums[i-1]){
            count++
        }
    }
    if count > 1 {
        return false
    }
    return count == 0 || nums[0] >= nums[len(nums) - 1]
}
